REQUIRE

```diff
@@ -1,9 +1,9 @@
 julia 0.4
-TikzPictures 0.3.2
+TikzPictures 0.3.4
```

Comment:
I assume the new version of `TikzPictures.jl`will be tagged as 0.3.4. This PR should not be merged until [this](https://github.com/sisl/TikzPictures.jl/pull/18) has been merged and tagged.